src/common/options.cc

```diff
@@ -3681,6 +3681,31 @@ std::vector<Option> get_global_options() {
     .set_default(true)
     .set_description(""),
 
+    Option("osd_mem_target", Option::TYPE_UINT, Option::LEVEL_BASIC)
```

Comment:
Do we want an _hdd and _ssd variant here, like we do with bluestore_cache_size?

Reply:
I'm not sure to be honest.  Potentially it's useful to be able to say that more memory should go to flash based OSDs vs HDD based OSDs, but I had a chance to watch users actually try to adjust the HDD/SSD/base config options and saw them adjusting the wrong settings (say changing the base value while an SSD override was already set) and confused why the cache size wasn't changing.  I wonder if we sort of need something like config templates for different device classes instead of the _SSD _HDD config options.